DIPs/DIP1012.md

```diff
@@ -96,86 +116,71 @@ Encompassed:
 
 Optionally encompassed:
 
-* final/virtual
-* Type_Info / Module Info generation (other components of -betterC?)
+* `final`/virtual
+* `export` -> `dynamicImport`/`dynamicExport`/`neither` 
```

Comment:
Pardon my Ignorance, what is planned for export?
Last I heard (albeit from Benjamins DConf 2016 talk) was that nobody was using export because there was no corresponding import and no way to switch between them. Benjamin suggested that making it an attribute would fix that. If there is something better planned I'm all ears.